### PR TITLE
Showcase companies using Got

### LINKED ISSUE #642: Showcase - Who uses Got?
We would like to highlight in the readme some companies/projects that use Got. 

Comment if you or your company use Got, and if possible, describe how you use it and anything else relevant. Include a link to the company/project website and your company/project logo too, if we're allowed to put it in the readme.

### PR DESCRIPTION
**Pull request by szmarczak**
Fixes #642

#### Checklist

- [x] I have read the documentation.
- [x] I have included a pull request description of my changes.
- [ ] I have included some tests.
- [ ] If it's a new feature, I have included documentation updates in both the README and the types.

### CODE REVIEW

**Comment by szmarczak** [comment]
Not sure whether we should include `MOXY` in the showcase, they're already at the very top of our README. It's up to you Sindre :)

File: readme.md [hunk 1]
```diff
@@ -48,6 +49,7 @@ For browser usage, we recommend [Ky](https://github.com/sindresorhus/ky) by the
 - [Plugins](documentation/lets-make-a-plugin.md)
 - [Used by 4K+ packages and 1.8M+ repos](https://github.com/sindresorhus/got/network/dependents)
 - [Actively maintained](https://github.com/sindresorhus/got/graphs/contributors)
+- [Trusted by many companies](#widely-used)
```
Comment (szmarczak) [inline readme.md:52]: Feel free to replace this with any text. I haven't thought of anything better yet :)

File: readme.md [hunk 2]
```diff
@@ -2315,6 +2317,63 @@ The Electron `net` module is not consistent with the Node.js `http` module. See
 - [Brandon Smith](https://github.com/brandon93s)
 - [Luke Childs](https://github.com/lukechilds)
 
+<a name="widely-used"></a>
+## These amazing comapnies are using Got
```
Comment (szmarczak) [inline readme.md:2321]:
```suggestion
## These amazing companies are using Got
```

File: readme.md (hunk 1 above)
Comment (sindresorhus) [reply readme.md:52]: Looks good to me.

**Comment by adityapatadia** [comment]
I am wondering how can we add our company name in GOT user?

**Comment by szmarczak** [comment]
Please comment at https://github.com/sindresorhus/got/issues/642 or open another issue :)

**Comment by szmarczak** [comment]
@sindresorhus Is it possible to increase vertical spacing a bit?

**Comment by sindresorhus** [comment]
@szmarczak I wanted that too, but I'm not sure how to do that while still letting the logos flow.

**Comment by szmarczak** [comment]
I don't think GitHub supports this at all...

**Pull request merged**